backend/src/modules/idim-webservice/idim-webservice.dto.ts

```diff
@@ -12,7 +13,7 @@ export class IDIRUserResponse {
     userId: string;
 
     @ApiProperty()
-    displayName: string;
+    guid: string;
```

Comment:
we don't need display name, and add the guid to the return data, so later we can store the guid when create a user.